### ISSUE TITLE
The .md URL can't be resolved correctly

### ISSUE THREAD

**Issue by VimWei** [vimwiki/vimwiki#1058]
Prior to submitting a new issue make sure to complete these steps:

- [ ] Checkout the `dev` branch and confirm the issue is present there as well.
      The `dev` branch contains fixes that may not have been merged to `master` yet.

- [x] Post the syntax you are using (default/mediawiki/markdown) **and** your vimwiki settings from your `.vimrc`

```
    Plug 'vimwiki/vimwiki', { 'commit': 'c9e6afe' }

    " def wiki dict
    let wiki = {}
    let wiki.name = 'ChenWei VimwikiMd'
    let wiki.path = 'd:/WeirdData/VimwikiMd/'
    let wiki.ext = '.md'
    let wiki.syntax = 'markdown'
    let wiki.nested_syntaxes = {'python': 'python'}
    let wiki.links_space_char = '_'
    let wiki.list_margin = 0
    let wiki.auto_toc = 1
    let wiki.auto_tags = 1
    let wiki.auto_generate_tags = 1

    " def vimwiki_list and so on.
    let g:vimwiki_list = [wiki]
    let g:vimwiki_ext2syntax = {'.md': 'markdown'}
    let g:vimwiki_global_ext = 1
    let g:vimwiki_autowriteall = 1
    let g:vimwiki_auto_chdir = 1
    let g:vimwiki_folding = 'expr'
```

- [x] Provide a detailed description of the problem including **steps to reproduce the issue**.

Enter the following URL： `- [Schema.yaml](https://github.com/LEOYoon-Tsaw/Rime_collections/blob/master/Rime_description.md)`

The result URL  is missing `.md` : `https://github.com/LEOYoon-Tsaw/Rime_collections/blob/master/Rime_description`

- [x] Include the output of `:VimwikiShowVersion`.

Os: Windows
Vim: 802
Branch: HEAD
Revision: c9e6afe
Date: 2020-05-26 23:09:58 +0200

**Comment by thecliguy** [comment]
@VimWei I've done some debugging and this issue should have been resolved by commit 068a26a. 

Please checkout the dev branch and confirm if the you're still experiencing the problem or not.

**Comment by VimWei** [comment]
Thanks!
